lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -54,6 +55,10 @@
     Strong
   }
 
+  private static final SparseArray<LottieComposition> STRONG_RAW_REF_CACHE = new SparseArray<>();
+  private static final SparseArray<WeakReference<LottieComposition>> WEAK_RAW_REF_CACHE =
+      new SparseArray<>();
+
   private static final Map<String, LottieComposition> STRONG_REF_CACHE = new HashMap<>();
```

Comment:
Can you rename this to `STRONG_ASSET_REF_CACHE` and `WEAK_ASSET_REF_CACHE`?

Reply:
👍